README.md

```diff
@@ -15,6 +15,7 @@ Then enable the sink using `WriteTo.Debug()`:
 ```csharp
 Log.Logger = new LoggerConfiguration()
     .WriteTo.Debug()
+    .MinimumLevel.Debug()
```

Comment:
have been looking at the serilog readme, and the `MinimumLevel` is not mentioned there either.

Space in there is at a premium, so even though it might trip quite a few people up, there's a very high bar for making the readme longer.

Also conflicted about whether I'd put this line here as it might suggest to people some linkage between this Sink being called Debug and the min level being Debug ?